### PR TITLE
fix(hammer.js): fix hammer class properties

### PR DESCRIPTION
**Pull request by daybrush**
#6

### CODE REVIEW

File: src/hammer.js [hunk 1]
```diff
@@ -20,7 +69,62 @@ export default class Hammer {
    * @const {string}
    */
 	static VERSION = "#__VERSION__#";
-
+	static DIRECTION_HORIZONTAL = DIRECTION_HORIZONTAL;
```
Comment (jongmoon) [inline src/hammer.js]: @younkue it's duplicated

File: rollup.config.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ export default [
 		},
 	},
 	{
-		input: "src/index.umd.js",
+		input: "src/Hammer.js",
```
Comment (jongmoon) [inline rollup.config.js:24]: @younkue Doesn't it need to apply in minified target? and Is `index.umd.js` useless anymore?

**Comment by jongmoon** [comment]
Constants exported in `index.js` is needed all?

File: rollup.config.js (hunk 2 above)
Comment (daybrush) [reply rollup.config.js:24]: @jongmoon  Hammer.js is same as index.umd.js. removed index.umd.js

**Review by jongmoon — APPROVED** [review]
LGTM

**Pull request merged**